src/modules/ens/sagas.ts

```diff
@@ -304,6 +309,25 @@ export function* ensSaga(builderClient: BuilderClient, ensApi: ENSApi) {
     }
   }
 
+  function* handleSetENSAddressRequest(action: SetENSAddressRequestAction) {
+    const { ens, address } = action.payload
+    try {
+      const wallet: Wallet = yield getWallet()
+      const signer: ethers.Signer = yield getSigner()
+
+      const nodehash = namehash(ens.subdomain)
+      const resolverContract = ENSResolver__factory.connect(ENS_RESOLVER_ADDRESS, signer)
+
+      const transaction: ethers.ContractTransaction = yield call(() => resolverContract['setAddr(bytes32,address)'](nodehash, address))
```

Comment:
There are lots of parts if the Builder where `call` is wrongly used 😢 . 
```suggestion
      const transaction: ethers.ContractTransaction = yield call([resolverContract, 'setAddr(bytes32,address)'], nodehash, address)
```